src/components/Pagination/Pagination.tsx

```diff
@@ -105,7 +105,7 @@ const Pagination: React.FC<PaginationProps> = ({
     onSkipBackward
 }) => {
     const isFirstPage = value === 1;
-    const isLastPage = value === Math.ceil(totalItems / pageSize);
+    const isLastPage = value === Math.ceil(totalItems / pageSize) || !totalItems;
```

Comment:
Nit: I'd suggest to make the new totalItems check a little more explicit, by either checking specifically against 0 (i.e. `totalItems === 0;`) or just defaulting to 1 (i.e. `pageSize) || 1;`).

Reply:
`totalItems === 0` would not cover cases like `null` o `undefined`.  `totalItems` should be a number type anyway so shall we be concerned about it?

`value === (Math.ceil(totalItems / pageSize) || 1);` I'm not sure is more readable, I find it less explicit then saying "is last page if we don't have total items"

Which do you this we should go for? 🙂

Reply:
What about dumb `(totalItems > 0 && pageSize > 0) ? Math.ceil(...) : false`

We allow any number, so potentially we can expect NaN and Infinity 🤷‍♂️ Let's say the behavior is not defined for that cases, but for all finite numbers I think we can check